spec/win_checker_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/board'
+require './lib/player'
+require './lib/space'
+require './lib/computer'
+require './lib/win_checker'
+require 'rspec'
+
+
+describe WinChecker do
+
+  it "exists" do
+    board = Board.new
+    player = Player.new(board)
+    computer = Computer.new(computer)
+    game_status = WinChecker.new(board, player, computer)
+    expect(game_status).to be_an_instance_of(WinChecker)
+  end
+
+
```

Comment:
We can probably hard code a win scenario here, in order to test each of the check_win methods. 

Effectively we can create a winning board by calling @board.board[row6][0].symbol == 'X' (4 times to create a win)
and then we can expect win_checker.check_horizontal_wins to equal true. 

The same could be done to simulate a vertical win and a horizontal win